### DISCUSSION TITLE
produced configure is broken in a fancy way

### DISCUSSION THREAD

**Discussion by robang74** [rauc/rauc#1032] [Show and tell]
Hi, this is my first try to compile RAUC. I did it on Ubuntu 22.04 LTS. The configure file generated is broke in a very fancy way. In a place is expected to have a ( but a newline instead, then the ( appears on a void line. So the script is broken in two points but nothing serious. The diff between the broken and the fixed configure files is reported here below

```
--- configure.broken	2022-12-29 08:18:22.297803789 +0100
+++ configure.fixed	2022-12-29 08:19:17.246251003 +0100
@@ -7271,8 +7271,7 @@ for as_dir in $PATH
 do
   IFS=$as_save_IFS
   case $as_dir in #(((
-    ''
- as_dir=./ ;;
+    '') as_dir=./ ;;
     */) ;;
     *) as_dir=$as_dir/ ;;
   esac
@@ -8170,7 +8169,6 @@ else
   am__fastdepCC_FALSE=
 fi
 
-)
 
 for ac_prog in catchsegv
 do
```

**Comment by Bastian-Krause** [comment]
What version of autoconf and m4 are you using? There have been some backward incompatibility notices for recent releases.

**Reply by robang74** [reply]
Here you are:

autoconf/jammy,jammy,now 2.71-2 all
m4/jammy,now 1.4.18-5ubuntu2 amd64

Thanks for the quick reply, R-

**Reply by ejoerns** [reply]
I cannot reproduce this with ubuntu 22.04 / autoconf 2.71

The macro that emits garbage here seems to be `AC_CHEKC_PROGS`

The issue however sounds close to what's described here https://www.winehq.org/pipermail/wine-devel/2020-October/176324.html or here https://github.com/varnishcache/varnish-cache/issues/3491 for autoconf 2.70
